README.md

```diff
@@ -34,12 +37,81 @@ Installation
 ember install ember-sinon-qunit
 ```
 
-
 Usage
 ------------------------------------------------------------------------------
 
-Import Ember-Sinon-QUnit's `test` method into your tests in place of Ember-QUnit's test. This creates a Sinon `sandbox`
-around that test via Sinon's `test` API. Then, you can access the following Sinon functions via `this` within the test callback:
+To use, import the setup method into your `tests/test-helper.js` file and execute it.
+
+```js
```

Comment:
Should we just show the full test-heler.js contents here? It's not very big, and sometimes it's nice to see code in context.